index.html

```diff
@@ -0,0 +1,251 @@
+                        creating a positive change with people from over 
+                        80 countries is taking place in October, in Korea.
+                    </p>
+                </article>
+                <article class="header-element third">
+                    <h3>2023.10.15(THU) ~ 16(FRI)</h3>
+                    <h4>
+                        @ Kenya National Institute of Conference (KICC), at Nairobi.
+                    </h4>
+                </article>
+            </article>
+            
+        </article>
+    </header>
+    
+    <section class="program_container">
+      <article class="container program-holder">
+        <article class="program_header">
+            <h3 class="h-title">
+                Main Program
+                <a href="#" class="text-border"></a>
+            </h3>
+        </article>
+       
+        <article class="progam_content">
+            <div class="item_content">
+                <a href="#"><img src="./assets/images/program_icon_01 1.svg" alt="program-icon"></a>
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from 
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+            <div class="item_content">
+                <a href="#"><img src="./assets/images/program_icon_02 1.svg" alt="program-icon"></a>
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from 
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+            <div class="item_content">
+                <a href="#"><img src="./assets/images/program_icon_03 1.png" alt="program-icon"></a>
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from 
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+            <div class="item_content">
+                <a href="#"><img src="./assets/images/program_icon_04 1.svg" alt="program-icon"></a>
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from 
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+            <div class="item_content">
+                <a href="#"><img src="./assets/images/program_icon_05 1.svg" alt="program-icon"></a>
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from 
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+        </article>
+        <article class="program_footer">
+            <h5>SEE THE WHOLE PROGRAM</h5>
+        </article>
+      </article>  
+    </section>
```

Comment:
- You have done well so far, but please remember to stick to the design template in terms of layout, spacing, borders, font color, background color, etc. The background color for this section is not the same as the given design template. Please fix this for both mobile and desktop versions🙏 

- In the mobile version, the section marked in the image below is not styled as it is in the given design template, please fix this 🙏. It is important to replicate the design template to give confidence to recruiters in our ability 👍 
Your project:
<img width="291" alt="sfjwrsgwr" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/4ebe10c0-6474-46d3-889e-0a55df857342">

Design template:
<img width="332" alt="rergerges" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/13f1f46f-ec8e-4979-a84d-a47e805b281f">